package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@uniswap/sdk",
+  "name": "uniswap-fuse-sdk",
   "license": "MIT",
-  "version": "3.0.3",
+  "version": "3.0.3a4",
```

Comment:
not sure how to handle the versions here..
lets keep the same version as the uniswap team - "3.0.3"